src/main.cpp

```diff
@@ -2940,7 +2969,7 @@ bool static DisconnectTip(CValidationState &state, bool fBare = false) {
     int64_t nStart = GetTimeMicros();
     {
         CCoinsViewCache view(pcoinsTip);
-        if (DisconnectBlock(block, state, pindexDelete, view, fAddressIndex) != DISCONNECT_OK)
+        if (DisconnectBlock(block, state, pindexDelete, view, true) != DISCONNECT_OK)
```

Comment:
The last argument changed to `updateIndices` because it applies to both addressindex and spentindex.